ds/beis_indicators/patents/make_patents.py

```diff
@@ -0,0 +1,178 @@
+#Script to make HESA indicators
+
+import csv
+import zipfile
+import io
+import os
+import requests
+import beis_indicators
+import numpy as np
+import json
+from data_getters.labs.core import download_file
+import random
+from ast import literal_eval
+
+from beis_indicators.utils.dir_file_management import *
+
+project_dir = beis_indicators.project_dir
+
+#Make directories
+make_dirs('patents',['raw','processed','interim'])
+                        
+def flatten_list(a_list):
+    
+    return([x for el in a_list for x in el])   
+
+#Download data from nesta data getters
+def patent_download(file_path=None, progress=True):
+    """ Fetch patent data
```

Comment:
I think else where we're using a single quote, I'd eventually use that for consistency